fdms/computation/country/eurostat.py

```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+from fdms.config.variable_groups import HICP
+import logging
+
+logging.basicConfig(filename='error.log', format='%(asctime)s %(module)s %(levelname)s: %(message)s',
+                    level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
Lines 3 to 9 can be replaced by just two blank lines if we are not using logging?